constants.py

```diff
@@ -685,3 +685,29 @@
 
 kVelocityControlMotorType = DCMotor.falcon500()
 kVelocityControlkV = 0.01
+
+# Intake Mechanism, need to replace values
+kIntakeCANID = 60
+kIntakeName = "IntakeMotor"
+kIntakePIDSlot = 0
+kIntakePGain = 0.12
+kIntakeIGain = 0
+kIntakeDGain = 0
+kIntakeRotationAmount = 3000
+
+kPivotCANID = 61
+kPivotName = "PivotMotor"
+kPivotPGain = 0.7
+kPivotIGain = 0
+kPivotDGain = 0
+
+kIntakeInverted = False
+kPivotInverted = False
+
+kPivotEncoderID = 0
```

Comment:
ik its temp but id 0 is the rio
make it reasonable because we change the CAN IDs to match whats written in code, not other way around